src/main/java/org/blockjam/core/config/ConfigManager.java

```diff
@@ -35,6 +33,8 @@
 import java.net.URL;
 import java.nio.file.Files;
 
+import static com.google.common.base.Preconditions.checkNotNull;
```

Comment:
Odd, not sure what that's about - the written specifications states that static imports precede non-static ones. We're also not strictly following Google style - we're following a derivative of it.